tokio-tls/examples/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
```

Comment:
What's the reason we need an extra crate?

Reply:
I think you are right, I removed it. Thanks :+1: